stories/Header.stories.js

```diff
@@ -81,6 +81,7 @@ const platformMockValue = {
             ],
             permissions: {
                 download_logs: true,
+                view_logs: true,
```

Comment:
It would be useful to mention view logs somewhere in Limani Story book, similar to existing Download Logs.

Reply:
I added a brief description, lmk what you think.